apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -0,0 +1,688 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+import { BootstrapTableOptions } from 'bootstrap-table';
+
+declare global {
+  interface Window {
+    popoverSubmitViaAjax: (e: any, popover: JQuery) => void;
+  }
+}
+
+declare global {
+  interface JQuery {
+    bootstrapTable(options: BootstrapTableOptions): JQuery;
+    bootstrapTable(method: string, ...parameters: any[]): JQuery | any;
+  }
+}
+
+interface AssessmentInstanceRow {
```

Comment:
It feels like this is possibly duplicated from somewhere? It'd be great to reuse whatever type we're using on the server.

Reply:
Updated to use the `AssessmentInstanceRow` from the server in [7efdf17](https://github.com/PrairieLearn/PrairieLearn/pull/9725/commits/7efdf1794fea6c432e21d53c25ae86cac0585ebd).